runtime/tutor.txt

```diff
@@ -415,6 +422,303 @@ _________________________________________________________________
+ respectively.
+
+ 1. Press C-s somewhere.
+ 2. Move far away in the file.
+ 3. Press C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                      SEARCHING IN FILE                        =
+=================================================================
+
+ Press / to search forward in file, enter to confirm search.
+ Press n to go to the next search match.
+ Press N to go to the previous search match.
+
+ 1. Press / and type in a common word, like 'banana'.
+ 2. Press enter to confirm the search.
+ 3. press n and N to cycle through the matches.
+
+ Like select mode, searching also supports regex!
+
+ Note: To search backwards, press ? (shift-/).
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press C-s to save position to the jumplist.
+   * Press C-i and C-o to go forward and backward in the jumplist.
+
+ * Press / to search forward in file, and ? to search backwards.
+   * Press n and N to cycle through search matches.
+
+
+
+
+
+=================================================================
+=                         JOINING LINES                         =
+=================================================================
+
+ Press J to join lines in selection.
+ This is a lesser used command, but is sometimes quite handy!
+
+ 1. Move the cursor to the line below marked -->.
+ 2. press x three times to select the other lines.
+ 3. press J to join the lines together.
+
+ --> This line sentence
+is spilling over
+onto other lines.
+
+
+
+
+
+=================================================================
+=                        INDENTING LINES                        =
+=================================================================
+
+ Press > to indent a line and < to unindent it
+
+ 1. Move the cursor to the line below marked -->.
+ 2. indent/unindent the lines with < and >, until they are uniform.
+
+ --> These lines
+    are indented
+        very poorly.
+
+ --> These lines
+     are indented
+     much better.
+
+
+
+
+
+=================================================================
+=                         OPENING LINES                         =
+=================================================================
+
+ Press o to add a newline and insert below the cursor.
+ Press O to add a newline and insert above the cursor.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Press o to open a line below and type your answer.
+
+ --> What is the best editor?
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ *  Press J to join lines within selection.
```

Comment:
```suggestion
 * Press J to join lines within selection.
```

Reply:
> The gaps separating the sections were intentional; they were such that there is an even amount of space between each section header (regardless of how much content is in the section) so that in an 80x24 terminal, page-up and page-down will smoothly switch between sections, keeping headers right at the top of the screen.

Alright I have largely resolved this but the section "copying and pasting text" is too big to fit on the screen. I'm thinking removing the "Note:" part of the footnotes or splitting this into two sections: 1 for yanking and 1 for pasting, what would you suggest?